wurst/_wurst/ErrorHandling.wurst

```diff
@@ -8,10 +8,16 @@ import GameTimer
 import MagicFunctions
 import Hashtable
 
-constant MUTE_ERROR_DURATION = 60
+constant MUTE_ERROR_DURATION = 5
```

Comment:
I don't think this is a good idea. Should probably make the value @configurable though